README.md

```diff
@@ -65,6 +65,39 @@ customElements.define('new-element', NewElement);
 
 See examples in the demo or try this live [JS bin](https://jsbin.com/zitotejimi/edit?html,output).
 
+## Uprading from 3.x
+
+The component has been migrated to LitElement and uses TypeScript now. This migration introduced two breaking changes.
+
+### Removed Polymer-specific `selection-changed` event used for 2-way bindings
+
+There were two events that notify about chart selection changes: `google-chart-select` and the Polymer-generated `selection-changed`.
+For consistency with other events (e.g. `google-chart-ready`), we keep only `google-chart-select`.
```

Comment:
"The Polymer-specific `selection-changed` event commonly used for 2-way binding has been removed. There were previously two events for observing chart selection changes: `google-chart-select` and the Polymer-generated `selection-changed`. For consistency with other events (e.g. `google-chart-ready`), we keep only `google-chart-select`."

I think that we can drop some of the details here.

Reply:
I used the suggestion. I'd like to keep the part explaining that the two events are equivalent, this makes the upgrade safer because it can be done in two steps:
1. update event used for data binding
2. upgrade dependency